### PR TITLE
homework

### PR DESCRIPTION
**Pull request by askulikov**

### CODE REVIEW

File: src/arsen.kulikov/homework.spec.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import { sum } from './homework';
+
+import * as _ from './homework';
```
Comment (LimarenkoDenis) [inline src/arsen.kulikov/homework.spec.ts]: usually  _ means lodash object, 
pls import via `import { sum, anotherFunc, ... }` syntax

File: src/arsen.kulikov/homework.spec.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+const salaries = {
```
Comment (LimarenkoDenis) [inline src/arsen.kulikov/homework.spec.ts]: pls declare const in specific test scope

File: src/arsen.kulikov/homework.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  = (obj: {[key:string]: any}): object => {
+    for (let key in obj) {
+      if (isNumeric(obj[key])) obj[key] = obj[key] * 2;
+    }
+    return obj;
+  }
+
+export const countBy: (a: number, b: number) => number[]
+  = (a: number, b: number): number[] => {
+    if (a <= 0 || b <= 0) throw new Error('Arguments must be a positive numbers');
```
Comment (LimarenkoDenis) [inline src/arsen.kulikov/homework.ts]: why only positive numbers?

Comment (askulikov) [reply src/arsen.kulikov/homework.ts]: Thank you for revise! Agree with another comments, about underscore, i think about 'unusable variable', and it's really not this case. My bad :)

This error throws with regard to notation in task: 

> Assume both the given number and the number of times to count will be positive numbers greater than 0.

**Pull request closed without merging**